### PR TITLE
Add XSS examples.

### PR DESCRIPTION
**Pull request by HelenaEksler**
#89

### CODE REVIEW

**Comment by HelenaEksler** [comment]
@amitaibu Ready.

**Comment by amitaibu** [comment]
Test is failing

**Comment by HelenaEksler** [comment]
@amitaibu Fixed.

File: app/templates/skeleton/modules/custom/skeleton_migrate/csv/user/user.csv [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Unique ID,Company,Username,Password,Email
 user1,company1|company2|company3,demo,1234,user1@example.com
-user2,company1|company2,Mickey,1234,user2@example.com
+user2,company1|company2|company0,Mickey,1234,user2@example.com
```
Comment (amitaibu) [inline app/templates/skeleton/modules/custom/skeleton_migrate/csv/user/user.csv]: what is `company0` ?

**Comment by amitaibu** [comment]
Can you please change the logo to be something that indicates XSS. Maybe this:

xss

**Comment by amitaibu** [comment]
@HelenaEksler ^^

**Comment by HelenaEksler** [comment]
@amitaibu 5 minutes

**Comment by HelenaEksler** [comment]
@amitaibu Done.

**Comment by amitaibu** [comment]
Thanks. Released `v1.3.0`

**Pull request merged**